<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>896</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(45, 45, 45);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>999999</width>
       <height>70</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>30</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: rgb(70,70,70);
	font: 30pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255,255,255);
	border-radius:10px;
}</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(55,55,55);
border-radius: 10px;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>70</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="p3_firstIndicator_comboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox {
        font: 8pt &quot;MS Shell Dlg 2&quot;;
        border: 2px solid rgb(55,55,55);
        border-radius: 10px;
        color: rgb(255, 255, 255);
        padding-left: 20px;
        padding-right: 20px;
        background-color: rgb(70,70,70);
}

QComboBox:hover {
        border: 2px solid rgb(90,90,90);
}

QComboBox QAbstractItemView {
     border: 2pxsolid rgb(45, 45, 45);
    selection-background-color:rgb(45, 45, 45);
    color: #FFF
                  }
</string>
           </property>
           <item>
            <property name="text">
             <string>SMA (Simple Moving Average)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>EMA (Exponential Moving Average)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>WMA (Weighted Moving Average)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BOLL (Bollinger Band)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>VWAP (Volumen Weighted Average Price)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>TRIX (Triple Exponential Average)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SAR</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Volumen</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MACD (Moving Average Eonvergence Divergence)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RSI (Relative Strength Index)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>KDJ</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OBV (On-balance volume)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CCI (Commodity Channel Index)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>StochRSI (Stoch Relative Strength Index)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>WR (Williams %)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DMI (Directional Movement Index)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MTM (Momentum)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MVM (Ease of Movement)</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="p3_firstIndicatorOptions_lineEdit">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
        font: 8pt &quot;MS Shell Dlg 2&quot;;
        border-radius: 10px;
        border: 2px solid rgb(55,55,55);
        color: rgb(255, 255, 255);
        padding-left: 20px;
        padding-right: 20px;
        background-color: rgb(70,70,70);
}
QLineEdit:hover {
        border: 2px solid rgb(85,85,85);
}
QLineEdit:focus{
        border: 2px solid rgb(85, 170, 255);
}
</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="p3_mark_comboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox {
        font: 8pt &quot;MS Shell Dlg 2&quot;;
        border: 2px solid rgb(55,55,55);
        border-radius: 10px;
        color: rgb(255, 255, 255);
        padding-left: 20px;
        padding-right: 20px;
        background-color: rgb(70,70,70);
}

QComboBox:hover {
        border: 2px solid rgb(90,90,90);
}

QComboBox QAbstractItemView {
     border: 2pxsolid rgb(45, 45, 45);
    selection-background-color:rgb(45, 45, 45);
    color: #FFF
                  }
</string>
           </property>
           <item>
            <property name="text">
             <string>=</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&gt;</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&gt;=</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&lt;</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&lt;=</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="p3_sedondIndicator_comboBox">
           <property name="styleSheet">
            <string notr="true">QComboBox {
        font: 8pt &quot;MS Shell Dlg 2&quot;;
        border: 2px solid rgb(55,55,55);
        border-radius: 10px;
        color: rgb(255, 255, 255);
        padding-left: 20px;
        padding-right: 20px;
        background-color: rgb(70,70,70);
}

QComboBox:hover {
        border: 2px solid rgb(90,90,90);
}

QComboBox QAbstractItemView {
     border: 2pxsolid rgb(45, 45, 45);
    selection-background-color:rgb(45, 45, 45);
    color: #FFF
                  }
</string>
           </property>
           <item>
            <property name="text">
             <string>MA (Moving Average)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>EMA (Exponential Moving Average)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>WMA (Weighted Moving Average)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BOLL (Bollinger Band)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>VWAP (Volumen Weighted Average Price)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>TRIX (Triple Exponential Average)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SAR</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Volumen</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MACD (Moving Average Eonvergence Divergence)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RSI (Relative Strength Index)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>KDJ</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OBV (On-balance volume)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CCI (Commodity Channel Index)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>StochRSI (Stoch Relative Strength Index)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>WR (Williams %)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DMI (Directional Movement Index)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MTM (Momentum)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MVM (Ease of Movement)</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="p3_secondIndicatorOptions_lineEdit">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
        font: 8pt &quot;MS Shell Dlg 2&quot;;
        border-radius: 10px;
        border: 2px solid rgb(55,55,55);
        color: rgb(255, 255, 255);
        padding-left: 20px;
        padding-right: 20px;
        background-color: rgb(70,70,70);
}
QLineEdit:hover {
        border: 2px solid rgb(85,85,85);
}
QLineEdit:focus{
        border: 2px solid rgb(85, 170, 255);
}
</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTreeWidget" name="treeWidget">
           <property name="styleSheet">
            <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(55,55,55);
color: rgb(255, 255, 255);</string>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <widget class="QPushButton" name="p3_cancel_button">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>130</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(85, 170, 255);
border: 0px sold;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
        <widget class="QPushButton" name="p3_addRule_button">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>130</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(85, 170, 255);
border: 0px sold;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>Add rule</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
